raiden_contracts/tests/test_etherscan_verify.py

```diff
@@ -160,9 +160,13 @@ def test_etherscan_verify_already_verified():
         etherscan_api = api_of_chain_id[chain_id]
         m.post(
             etherscan_api,
-            text='{ "status": "0",'
-            '"result" : "Contract source code already verified",'
-            '"message" : "" }',
+            text="""
```

Comment:
From a quick ipython session it seems like you don't even need to escape the `"`:
```
In [1]: a = """
   ...: {"abc": "def"}
   ...: """

In [2]: a
Out[2]: '\n{"abc": "def"}\n'

In [3]:             text="""
   ...:             {
   ...:                 \"status\": \"0\",
   ...:                 \"result\" : \"Contract source code already verified\",
   ...:                 \"message\" : \"\"
   ...:             }
   ...:             """,

In [4]: text
Out[4]: ('\n{\n    "status": "0",\n    "result" : "Contract source code already verified",\n    "message" : ""\n}\n',)
```

Reply:
My bad, not even tried this.